tokio/src/net/addr.rs

```diff
@@ -58,17 +84,33 @@ impl sealed::ToSocketAddrsPriv for str {
     }
 }
 
+// ===== impl &'a [SocketAddr] =====
+
+impl<'a> ToSocketAddrs for &'a [SocketAddr] where ReadyFuture<Self::Iter>: 'a {}
```

Comment:
Hmm... this impl got [E0275 error](https://dev.azure.com/tokio-rs/Tokio/_build/results?buildId=3297&view=logs). (If I remove where clause, I got E0495 error.) So remove this impl for now.

Reply:
@taiki-e is there any intent towards adding back `impl &'a [SocketAddr]`? I was hoping to use it when attempting to bind on one of many SocketAddr. For now, I'm having to loop through each address manually, unless there's an alternative that I didn't see before.